knotx-core/src/main/java/io/knotx/repository/http/HttpRepositoryConnectorProxyImpl.java

```diff
@@ -144,12 +145,21 @@ private String encode(String value) {
   }
 
   private ClientResponse toResponse(Buffer buffer, final HttpClientResponse httpResponse) {
-    if (httpResponse.statusCode() >= 300 && httpResponse.statusCode() < 400) { //redirect responses
-      LOGGER.info("Repository 3xx response: {}, Headers[{}]", httpResponse.statusCode(),
+    if (HttpStatusClass.SUCCESS.contains(httpResponse.statusCode())) {
```

Comment:
For a better readability of each condition, get the `.statusCode()` into variable and do the check against that variable

Reply:
Done

Reply:
I think if it is not useful information to see the request path (especially in `warn` and `error` level) in the message.
What do you think?